dags/aps/parser.py

```diff
@@ -143,10 +143,9 @@ def _get_licenses(self, x):
             licenses = []
             for right in rights:
                 url = right["url"]
-                url_parts = url.split("/")
-                clean_url_parts = list(filter(bool, url_parts))
-                version = clean_url_parts.pop()
-                license_type = clean_url_parts.pop()
+                clean_url_parts = list(filter(bool, url.split("/")))
+                version = clean_url_parts[-1]
+                license_type = clean_url_parts[-2]
```

Comment:
what if it has only one `/`? This will throw an exception. Also this is APS why are we changing it in this PR?

Reply:
I just went throw the comments, and this one was the same solution as in IOP, [Parth's comment ](https://github.com/cern-sis/workflows/pull/52#discussion_r993481829)

 I will fix it on just IOP

Reply:
this file is on aps :)

Reply:
Yes, I understand. I can remove the fix, however, it was a good observation from Parth on IOP. And the same mistake on was done on APS. Should I revert the motdification?